src/clj_kondo/impl/analyzer.clj

```diff
@@ -1627,6 +1627,21 @@
             (analyze-children (ctx-with-bindings ctx binding)
                               forms-exprs))))
 
+(defn analyze--> [ctx expr]
+  (let [fn-node (fn fn-node [node]
+                  (or (fn-node? node)
+                      (and (list-node? node)
+                           (contains? '#{fn fn*} (:value (first (:children node)))))))
+        fn-nodes (filter fn-node (nnext (:children expr)))]
+    (run! (fn reg-node! [node]
+            (findings/reg-finding!
+             ctx
+             (node->line (:filename ctx)
+                         node
+                         :threaded-fn-form
+                         "threading a value into the name slot of an anonymous fn")))
```

Comment:
New lint warnings follow the following convention:

Sentences start with a capital letter